gitpod-io/gitpod
# [supervisor] add browser code telemetry tracking

## Description

Add browser vscode telemetry tracking

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/10929

## How to test

- Choose IDE code latest
- Open a workspace
- Scale down `ws-proxy` replicas to zero
- Wait about 1 minute, and check if conn lost, and track segment

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:
